lldb/test/API/functionalities/thread/finish-from-empty-func/TestEmptyFuncThreadStepOut.py

```diff
@@ -15,10 +15,23 @@ class FinishFromEmptyFunctionTestCase(TestBase):
     def test_finish_from_empty_function(self):
         """Test that when stopped at a breakpoint in an empty function, finish leaves it correctly."""
         self.build()
-        exe = self.getBuildArtifact("a.out")
-        target, process, thread, _ = lldbutil.run_to_name_breakpoint(
-            self, "done", exe_name=exe
+        target, _, thread, _ = lldbutil.run_to_source_breakpoint(
+            self, "// Set breakpoint here", lldb.SBFileSpec("main.c")
         )
+        # Find the last instruction address of 'done()' and set a breakpoint there.
```

Comment:
Include the reasoning from the PR description here like:
If we just breakpoint "done" then it will break on the epilogue. Instead we need the last instruction of the body (whatever the term is) of "done".